circuits/circom/membership.circom

```diff
@@ -2,6 +2,8 @@ pragma circom 2.0.2;
 
 include "node_modules/circom-ecdsa/circuits/zk-identity/eth.circom";
 include "node_modules/circom-ecdsa/circuits/ecdsa.circom";
+include "node_modules/circom-ecdsa/node_modules/circomlib/circuits/bitify.circom";
+include "node_modules/circom-ecdsa/node_modules/circomlib/circuits/comparators.circom"; // TODO: fix insane dependency twiddling
```

Comment:
I believe `comparators` already include `bitify`. That's why you are getting duplicate dependency error.

Reply:
🤔
I see that comparators does include bitify...
It doesn't cause an error locally though.
So I am not sure...